doc/bc/changes-5.4.md

```diff
@@ -189,6 +189,10 @@ Changes affecting version compatibility with former or future versions.
   when deleting last version of the Content. Since Content without a version does not make sense, in
   this case `eZ\Publish\Core\Repository\ContentService::deleteContent()` should be used instead.
 
+* 5.4.4: $fieldFilters argument on Search service has been renamed to $languageFilter
+  Reason is to better communicate what the argument is used for. No changes
+  to it's structure is done so this is a purly cosmetic change.
```

Comment:
Should for 5.4 have separate line describing that languages will be treated as a prioritized list of languages now.